### DISCUSSION TITLE
Writing to a file without removing metadata

### DISCUSSION THREAD

**Discussion by matthewauld** [beakerbrowser/beaker#1711] [General]
It seems `beaker.hyperdrive.writeFile` removes all the metadata on that file - is there a way to prevent that? I don't see anything in the docs, and it would be really handy to have a `preserveMetadata` option or something!

**Comment by pfrazee** [comment]
I'll check on this, I agree this is a big footgun. Can you open an issue?

**Comment by matthewauld** [comment]
Will do!
